src/test/java/cow/farm/utils/QuantityCheckingListTest.java

```diff
@@ -19,10 +16,10 @@ public void shouldReturnMapOfValues() {
 		list.add("b");
 		list.add("d");
 		//then
-		HashMap map = list.getQuantityOfEachObject();
-		Assert.assertEquals(map.get("a"), 2);
-		Assert.assertEquals(map.get("b"), 2);
-		Assert.assertEquals(map.get("c"), 1);
-		Assert.assertEquals(map.get("d"), 1);
+		//HashMap map = list.getQuantityOfEachObject();
```

Comment:
Prawdę mówiąc to testy powinny być na deczko wyższym poziomie, tylko podejście "per klasa" jest proste do ogarnięcia na początku. W tym drugim projekcie postaram się wprowadzić testy DSL, które ładnie oplotą kod. Ważne jest pokrycie testami. Jeżeli nie mamy testów takich end-to-end to pozostają takie bardziej przy klasie. Ale zgadzam się, powinniśmy pisać test per przypadek użycia (od interfejsu do interfejsu).